omc3/plotting/plot_amplitude_detuning.py

```diff
@@ -0,0 +1,350 @@
+"""
+Plot Amplitude Detuning Results
+------------------------------------
+
+Provides the plotting function for amplitude detuning analysis
+
+
+**Arguments:**
+
+*--Required--*
+
+- **kicks**: Kick files as data frames or tfs files.
+
+- **labels** *(str)*: Labels for the data. Needs to be same length as kicks.
+
+- **plane** *(str)*: Plane of the kicks.
+
+  Choices: ``('X', 'Y')``
+
+*--Optional--*
+
+- **action_plot_unit** *(str)*: Unit the action should be plotted in.
+
+  Choices: ``['km', 'm', 'mm', 'um', 'nm', 'pm', 'fm', 'am']``
+  Default: ``um``
+- **action_unit** *(str)*: Unit the action is given in.
+
+  Choices: ``['km', 'm', 'mm', 'um', 'nm', 'pm', 'fm', 'am']``
+  Default: ``m``
+- **correct_acd**: Correct for AC-Dipole kicks.
+
+  Action: ``store_true``
+- **detuning_order** *(int)*: Order of the detuning as int. Basically just the order of the applied fit.
+
+  Default: ``1``
+- **manual_style** *(DictAsString)*: Additional plotting style.
+
+  Default: ``{}``
+- **output** *(str)*: Save the amplitude detuning plot here.
+
```

Comment:
is this a directory or filename?

Reply:
filename. But it will be modified by the _dQxdJy etc. extension